db/migrate/20190514155447_create_ambassador_tasks.rb

```diff
@@ -0,0 +1,20 @@
+class CreateAmbassadorTasks < ActiveRecord::Migration
+  def change
+    create_table :ambassador_tasks do |t|
+      t.string :description, null: false, default: ""
+
+      t.timestamps null: false
+    end
+
+    create_table :ambassador_task_assignments do |t|
+      t.belongs_to :user, index: true, null: false
+      t.belongs_to :ambassador_task, index: true, null: false
+      t.boolean :completed, null: false, default: false
```

Comment:
Rather than a boolean completed, I have leaned toward a timestamp (so `completed_at`). If you don't like this, let me know - it just often becomes helpful down the road.